server/index.js

```diff
@@ -10,13 +10,18 @@ app.use(cors())
 
 logger.info(process.env.BASE_URL)
 
-app.use('/api', function(req, res) {
-    let startTime = new Date()
-    logger.info("Incoming " + req.method + " request")
-    logger.debug("request URL: " + req.url)
-    req.pipe(request(process.env.BASE_URL + req.url)).pipe(res);
-    let endTime = new Date()
-    logger.info("Response %s. Completed in: %dms", res.statusCode, (endTime - startTime))
+app.use('/api', function (req, res) {
+    if (req.url === '/redmineUrl') {
+        logger.info("Redmine URL sent to frontend")
+        res.send(process.env.BASE_URL)
+    } else {

```

Comment:
this is needed but as a separate endpoint take a look at this:
https://github.com/dil-lkobor/redmine-kanban/blob/kanban-filter/server/index.js#L13
keeping the `/api/something` format